content/en/docs/usecases/datadef.md

```diff
@@ -15,7 +15,7 @@ values, ranges, and various other constraints.
 OpenAPI even allows for complex logical combinators.
 
 A key difference, however, is that these standards do not unify schema
-and values, the thing that makes CUE so powerful.
+and values—the thing that makes CUE so powerful.
```

Comment:
I think the intention of this sentence would be slightly better phrased as:

> A key difference, however, is that these standards do not unify schema and values—one thing that makes CUE so powerful.

Because there are many things that makes CUE so powerful, this is one (albeit important) such thing.

Thoughts?

Reply:
It's probably a better question for the original author (@mpvl, perhaps?), but my reading of the original intent was that this particular feature of cue is singularly important and distinctive -- and as a relative newbie to CUE, that strikes me as both plausible and interesting.